### ISSUE TITLE
jade failed to install

### ISSUE THREAD

**Issue by tjwudi** [Automattic/wp-calypso#2931]
I got following error message when executing `make run` or `npm install`.

```
npm ERR! git rev-list -n1 29784fd: fatal: ambiguous argument '29784fd': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 29784fd: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 29784fd: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 29784fd: 
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 29784fd
npm ERR! fatal: ambiguous argument '29784fd': unknown revision or path not in the working tree.
npm ERR! Use '--' to separate paths from revisions, like this:
npm ERR! 'git <command> [<revision>...] -- [<file>...]'
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/John/Code/wp-calypso/npm-debug.log
```

It turns out that `29784fd` is the designated SHA of jade in **package.json**.

```
$ grep -r 29784fd .
./package.json:    "jade": "jadejs/jade#29784fd",
```

jade does have [this SHA](https://github.com/pugjs/jade/commits/29784fd) in history of the master branch.

I am using node 4.2.6 and npm 2.14.12.

**Comment by tjwudi** [comment]
Just tried npm 3. It still raised this error message, but instead it continued execution without halting, whilst npm 2 will actually quit installation.

**Comment by tjwudi** [comment]
Just tried npm 3, still failing.

**Comment by bluefuton** [comment]
Thanks for the report @tjwudi! Should be fixed by #2934.

**Comment by tjwudi** [comment]
Nice! Thanks!
